dbt/adapters/databricks/connections.py

```diff
@@ -105,6 +110,8 @@ def emit(self, record: logging.LogRecord) -> None:
 CLIENT_ID = "dbt-databricks"
 SCOPES = ["all-apis", "offline_access"]
 
+USE_LONG_SESSIONS = os.getenv("DBT_DATABRICKS_LONG_SESSIONS", "FALSE").upper() == "TRUE"
```

Comment:
Any reason to default to false?  I think if we're concerned about breaking behavior, we should release as a beta with defaulted to true.  If this change improves performance, as we expect, it would be annoying to make users opt in, unless we know if breaks some use cases.